### DISCUSSION TITLE
code error

### DISCUSSION THREAD

**Discussion by sarco-rdf** [ViktorSky/amino-coin-generator#2] [General]
[coins-generator][main-proccess][--------------@gmail.com]: Invalid Request. Please update to the latest version. If the problem continues, please contact us..

**Comment by ViktorSky** [comment]
update the script, download again

**Reply by Wispwee** [reply]
I also have this issue. On latest version 3.0.2. have re-downloaded. How do I fix?

**Reply by sarco-rdf** [reply]
3.0.2? The script in its latest published version is 3.0.1, however the error is temporary, the amino servers fail, but it currently works, not 100% but it works correctly, I have been using it for months and 90% of the errors have been due to amino crashes, you usually leave the program running and it solves itself, if 24 hours pass and it is not solved, restart the program

**Reply by Wispwee** [reply]
I thought that was perhaps the issue. I will try that, thank you.

**Reply by sarco-rdf** [reply]
Currently the script no longer works, amino team seems to have touched something that blocks the requests module or something
